### PR TITLE
Fixed using mysql-prefix

### LINKED ISSUE #85: option --mysql_db_prefix will not overwrite the default value j_
using official version 1.4.11
Using the option --mysql-db-prefix produces an error, That's fine for the moment, because the syntax is not allowed (yet). Using --mysql_db_prefix instead is accepted, but the new value is not used with the table names. They still are prefixed with the default "j_".

### PR DESCRIPTION
**Pull request by thuebschmann**
Fixes #85 

I changed that the givven prefix is use when the parameter set

### CODE REVIEW

File: src/Joomlatools/Console/Command/Database/AbstractDatabase.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         else
         {
-            $this->target_db_prefix = '';
+            $this->target_db_prefix = $input->getOption('mysql_db_prefix') ?: $input->getOption('mysql-db-prefix');
```
Comment (stevenrombauts) [inline src/Joomlatools/Console/Command/Database/AbstractDatabase.php:90]: @thuebschmann The code inside this `else`  block is only run when the user has explicitly set a database name. If we're overriding the database, it wouldn't make sense to use a different `target_db_prefix`: it should simply be set to an empty string. 

This is also mentioned in the option's description: https://github.com/joomlatools/joomlatools-console/blob/aaa04096a21402e827dbc2b1ac7d73704445ec56/src/Joomlatools/Console/Command/Database/AbstractDatabase.php#L59

File: src/Joomlatools/Console/Command/Database/Install.php [hunk 2]
```diff
@@ -115,7 +114,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         {
             $tmp      = tempnam('/tmp', 'dump');
             $contents = file_get_contents($import);
-            $contents = str_replace('#__', 'j_', $contents);
+            $contents = str_replace('#__',  $this->target_db_prefix, $contents);
```
Comment (stevenrombauts) [inline src/Joomlatools/Console/Command/Database/Install.php:117]: @thuebschmann I think you're confusing the *database* prefix with the *Joomla table* prefix here? :) The MySQL database prefix mentioned here is only used for one thing: naming the database.

The `j_` prefix is used for the Joomla *table* names, not the *database*. If you need this one to be changed, we can add a separate option for this, something like `--joomla-table-prefix`. Happy to accept a PR for this.

Comment (thuebschmann) [reply src/Joomlatools/Console/Command/Database/Install.php:117]: @stevenrombauts 
Yes I think an option for table prefix is the better solution.
So it is posible to install two or more joomla in one database. 
Can you add this obtion?

File: src/Joomlatools/Console/Command/Database/AbstractDatabase.php (hunk 1 above)
Comment (thuebschmann) [reply src/Joomlatools/Console/Command/Database/AbstractDatabase.php:90]: you are righrt
then we need the table prefix option

**Comment by stevenrombauts** [comment]
Closing this PR, but as discussed always happy to accept another PR to handle different Joomla table prefixes.

**Pull request closed without merging**